entity-framework/core/managing-schemas/migrations/applying.md

```diff
@@ -156,6 +159,76 @@ Note that this can be used to roll back to an earlier migration as well.
 
 For more information on applying migrations via the command-line tools, see the [EF Core tools reference](xref:core/cli/index).
 
+## Bundles
```

Comment:
So at least thinking like a clueless newcomer, this kinda begs the question of why we SQL scripts are still "the recommended way" to apply migrations (above). It may be good to add a point or two about where SQL scripts are still better (e.g. allow inspection of the SQL before application, etc.).

Reply:
That's why I left SQL scripts first in the document. Those points are made there and are still just as valid after the introduction of bundles.